.ci/pytorch/test.sh

```diff
@@ -105,13 +105,13 @@ if [[ "$TEST_CONFIG" == *crossref* ]]; then
   export PYTORCH_TEST_WITH_CROSSREF=1
 fi
 
-if [[ "$TEST_CONFIG" == *dynamo* ]]; then
-  export PYTORCH_TEST_WITH_DYNAMO=1
-fi
+# if [[ "$TEST_CONFIG" == *dynamo* ]]; then
```

Comment:
I am keeping these commented on purpose for when they return. Can also rm them if folks prefer. No strong opinion